libs/gdk/src/lib/gdk-form/gdk-form.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { GdkFormComponent } from './gdk-form.component';
+
+describe('GdkFormComponent', () => {
+  let component: GdkFormComponent;
+  let fixture: ComponentFixture<GdkFormComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [GdkFormComponent]
+    }).compileComponents();
```

Comment:
Test is failling, probably due to missing imports in testing module

Reply:
ok